unbxdSearch.js

```diff
@@ -1277,16 +1380,36 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
 
             return this;
         },
-        setDefaultParams: function(params) {
+        setDefaultParams: function (params) {
             var newparams, diff;
             var oldparams = JSON.stringify(params);
             this.options.setDefaultFilters.call(this);
+
+            this.params['extra']['version'] = 'V2';
```

Comment:
where are we using this `version`.

Reply:
This param is version = V2 , which we add for all customer integrations. It's for the facets.

<img width="801" alt="Screenshot 2020-06-05 at 9 50 40 PM" src="https://user-images.githubusercontent.com/47173038/83899719-8a825c00-a776-11ea-967f-af8a9165425f.png">

Reply:
This will be enabled in the backend by search brewer. We may not need this. Also, COM migrated customers will need `version=V1`.